spec/services/facility_stats_service_spec.rb

```diff
@@ -0,0 +1,119 @@
+require "rails_helper"
+
+RSpec.describe FacilityStatsService do
+  let(:small_facility) { create(:facility, name: "small1", facility_size: "small") }
+  let(:december) { Date.parse("1-12-2020").beginning_of_month }
```

Comment:
You are calling this let var `december`, but the period is for January.  

I would recommend `Date.parse("12-01-2019")` for this if you really want December.

Reply:
Whoops.